src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -0,0 +1,593 @@
+/*
+ * Copyright @ 2015 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.impl.neomedia.rtp.remotebitrateestimator;
+
+
+import org.ice4j.util.*;
+import org.jitsi.impl.neomedia.*;
+import org.jitsi.impl.neomedia.transform.*;
+import org.jitsi.service.neomedia.*;
+import org.jitsi.service.neomedia.rtp.*;
+
+import java.util.*;
+
+/**
+ * webrtc.org abs_send_time implementation as of June 26, 2017.
+ * commit ID: 23fbd2aa2c81d065b84d17b09b747e75672e1159
+ * @author Julian Chukwu
+ */
+public class RemoteBitrateEstimatorAbsSendTime
+    extends SinglePacketTransformerAdapter
+    implements RemoteBitrateEstimator
+{
+    //@Todo Ask for alternative to resolve import conflict between
+    //org.jitsi.util.* and org.ice4j.util.* when importing Logger
+    // and RateStatistics. For now, see below.
```

Comment:
Using the full name is fine. In these cases, as an exception, we import the specific class that we need (import org.jitsi.util.Logger).